rados/rados_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"io"
 	"io/ioutil"
 	"strconv"
+
```

Comment:
Let's get rid of both this line and the commented out import.

Reply:
The empty line is inserted by `goimports` so I suppose it is a common style. But I'm totally fine removing that. :slightly_smiling_face:

Reply:
The empty line has to be there to separate the imports for internal packages and vendered packages. Otherwise `make test-docker` would fail.